syntax = "proto3";
package ColmnarDB.NetworkClient.Message;

message SetDatabaseMessage {
    string DatabaseName = 1;
}